src/vivarium_gates_mncnh/constants/data_values.py

```diff
@@ -325,18 +320,18 @@ class __DeliveryFacilityTypes(NamedTuple):
 
 DELIVERY_FACILITY_TYPE_PROBABILITIES = {
     "Ethiopia": {
-        FACILITY_CHOICE.P_HOME_PRETERM: 0.38,
-        FACILITY_CHOICE.P_HOME_FULL_TERM: 0.55,
+        # FACILITY_CHOICE.P_HOME_PRETERM: 0.38,
```

Comment:
These need  to be commented out since we loop through them in artifact generation